lib/electric_slide/agent.rb

```diff
@@ -14,10 +14,10 @@ def initialize(opts = {})
       @presence = opts[:presence] || :available
     end
 
-    def presence=(new_presence)
```

Comment:
Should probably remove [access to `#presence=`](https://github.com/adhearsion/electric_slide/pull/45/files#diff-0a708c3adfc0f0b8531a6430e311ce3aR5) completely. Also needs a change log entry as this is a significant API change.